plugins/services/src/js/service-configuration/ServiceNetworkingConfigSection.js

```diff
@@ -120,7 +120,7 @@ class ServiceNetworkingConfigSection extends ServiceConfigBaseSectionDisplay {
                 prop: keys.port,
                 className: getColumnClassNameFn(),
                 render(prop, row) {
-                  return appDefinition.requirePorts
+                  return getDisplayValue(row[prop])
```

Comment:
can we set the `getDisplayValue` result to a variable here so we don't call the function twice? it is creating components and it seems wasteful to call it twice.

Reply:
Will do.

Reply:
Implemented a slightly more concise solution.